protocol/src/test/kotlin/models/CloseTest.kt

```diff
@@ -21,19 +23,20 @@ class CloseTest {
   @Test
   fun `sign populates close signature`() {
     val close = TestData.getClose()
-    close.sign("fakepk", "fakekid")
+    close.sign(TestData.ALICE_DID)
 
-    assertThat(close.signature).isEqualTo("blah")
+    assertThat(close.signature).isNotNull()
+    assertThat(JWSObject.parse(close.signature))
```

Comment:
is this `assertThat` checking the JWSObject.parse() does not throw an exception? or something else? usually this is daisy-chained with `.isXYZ()`

Reply:
aye `parse` walidates